docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1039,6 +1048,13 @@ As the core components use emotion as a styled engine, the props used by emotion
 
 [This codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#variant-prop) will automatically update your code.
 
+- Remove the `labelWidth` prop because it's not longer need.
```

Comment:
```suggestion
- Remove the `labelWidth` prop. The `label` prop now servers the same purpose, using CSS layout instead of JavaScript measurement to render the gap in the outlined.
```